app/models/infra_conversion_job/dispatcher.rb

```diff
@@ -1,7 +1,7 @@
 class InfraConversionJob
   class Dispatcher < Job::Dispatcher
     def self.waiting?
-      job_class.where.not(:state => %w[finished waiting_to_start])
+      job_class.where.not(:state => %w[finished waiting_to_start]).any?
```

Comment:
😍

Reply:
Yeah.....[[this]](https://github.com/ManageIQ/manageiq/pull/21340/files#diff-394a3e04595a24d168a9a964670bac5c52f6a4bd36a8062c346f059ce8cb3472L192) was always truthy so the job proxy dispatcher was always running :cry: